perun-ldapc/src/main/resources/log4j.properties

```diff
@@ -1,39 +0,0 @@
-# log4j Configuration
```

Comment:
I have introduced cz.metacentrum.perun.logging.PerunLogbackConfigurator into perun-base module which gets executed if no logback.xml is found on classpath. In that case it tries to load ${perun.conf.custom/logback.xml, /etc/perun/logback.xml and classpath:logback-default.xml in that order. Thus if a module (like perun-ldapc) contains its own logback.xml, it takes precedence.